_FIPS201/authentication.md

```diff
@@ -206,8 +206,7 @@ The symmetric card authentication key and associated SYM-CAK authentication mech
  If the symmetric card authentication key is present, it **SHALL** be used for PIV cardholder authentication using the following
 steps:
 
-- The CHUID, PIV authentication certificate, or card authentication certificate data element is read
-    from the PIV Card and is checked to ensure that the card has not expired.
+- The CHUID or another data element[^expired2] is read from the card. The signature of the CHUID or another data element is verified to ensure that the card has not expired and that the card comes from a trusted source.
```

Comment:
```suggestion
- The CHUID or another data element[^expired2] is read from the card and is checked to ensure that the card has not expired.
```


```suggestion
- The CHUID or another data element[^expired2] is read from the card. The signature of the CHUID or another data element is verified to ensure that the card has not expired and that the card comes from a trusted source.
```

The following bullets talks about signature checks and 'trusted source' .  That is why I removed it here.